static/src/js/components/Map/Map.scss

```diff
@@ -133,6 +133,113 @@ button.edsc-map-spatial-drawing__button--cancel {
   }
 }
 
+.edsc-map-layer-switcher {
+  z-index: 15;
+  color: utils.$color__carbon--70;
+  background-color: utils.$color__spacesuit-white;
+  border-radius: 2px;
+  padding: 0;
+  overflow: visible;
+  font-weight: 500;
+  width: 1.75rem;
+  height: 1.75rem;
+
+  button {
+    width: 1.75rem;
+    height: 1.75rem;
+    background-color: transparent;
+    border: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    
+    svg {
+      width: 1rem;
+      height: 1rem;
+      margin-left: -1px;
+    }
+  }
+  
+  &__panel {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    background-color: utils.$color__spacesuit-white;
+    border-radius: 0.25rem;
+    opacity: 0;
+    visibility: hidden;
+    width: 18rem;
+    color: utils.$color__carbon-black;
+    text-shadow: none;
+    z-index: 16;
+
+    &--visible {
+      opacity: 1;
+      visibility: visible;
+    }
+  }
+
+  &__options {
+    padding: 0.75rem 0.75rem 0.5rem;
+ 
+    & > div:nth-child(3) {
```

Comment:
Hard coding this to show up after the 3rd element feels weird to me. Is there an easier way to just put an `<hr />` after the last base layer instead?